pkg/sentry/pgalloc/pgalloc.go

```diff
@@ -775,7 +786,12 @@ func (f *MemoryFile) forEachMappingSlice(fr memmap.FileRange, fn func([]byte)) e
 	return nil
 }
 
-func (f *MemoryFile) getChunkMapping(chunk int) ([]uintptr, uintptr, error) {
+func (f *MemoryFile) getChunkMapping(chunk int, populate bool) ([]uintptr, uintptr, error) {
+	var flags int
+	if populate {
+		flags |= unix.MAP_POPULATE
+	}
```

Comment:
flags really is only optionally holding populate flag. I think it makes more sense for this to be something like:

```
flags := uintptr(unix.MAP_SHARED)
if populate {
    flags |= uintptr(unix.MAP_POPULATE)
}
...
unix.Syscall6(..., flags,...)
```

Reply:
ok, done